configure-client.html

```diff
@@ -9,359 +9,521 @@
+        </div> <!-- end #line-block -->
 
-</li>
+    </li>
 
-<li><p class="first">Navigate to the <span class="bold">Contents</span> folder.</p>
-<p>Select the <span class="bold">Java</span> folder.</p>
-<p>Drag and drop, or copy, the third-party <span class="code-example">.jar</span> file into the Java folder.</p>
+    <li><p class="first">
+            Navigate to the <span class="bold">Contents</span>
+            folder.
+        </p>
+        
+        <p>Select the <span class="bold">Java</span> folder.</p>
+        
+        <p>
+            Drag and drop, or copy, the third-party <span
+            class="code-example">.jar</span> file into the Java
+            folder.
+        </p>
 
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
+        <div class="line-block">
+            <div class="line"><br /></div> <!-- end #line -->
+        </div> <!-- end #line-block -->
 
-<img alt="../images/configureClientJava.jpg" class="align-center" src="images/configureClientJava.jpg" style="width: 80%;" />
+        <img alt="../images/configureClientJava.jpg" class="align-center"
+        src="images/configureClientJava.jpg" style="width: 80%;" />
 
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
+        <div class="line-block">
+            <div class="line"><br /></div> <!-- end #line -->
+        </div> <!-- end #line-block -->
 
-</li>
+    </li>
 
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
+    <div class="line-block">
+        <div class="line"><br /></div> <!-- end #line -->
+    </div> <!-- end #line-block -->
 
-<li><p class="first">Restart OMERO.insight to enable the new JAR’s functionality.</p>
+    <li><p class="first">
+            Restart OMERO.insight to enable the new JAR’s
+            functionality.
+        </p>
 
-</li>
+    </li>
 
 </ol>
 
 <div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
+    <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png"
+    alt="Image for link to top of page" style="float: right;"/></a></p>
 
 <h2><a class="anchor" id="jarwindows"></a>OMERO.insight on Windows</h2>
 
 <div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
+    <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
 <ol>
 
-<li><p class="first">Locate the <span class="bold">OMERO.insight.app</span> in Explorer.</p>
-<p>Open the <span class="bold">libs</span> directory.</p>
-<p>Drag and drop, or copy, the third-party <span class="code-example">.jar</span> file into the libs folder.</p>
+    <li><p class="first">
```

Comment:
Minor point: in a few places like this, it would be a bit neater to have the ```<p>``` on a new line. Could search and replace ```<li><p```